.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  publish:
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+      - name: Install dependencies
+        run: npm install
+      - name: Set registry
+        run: 'echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc'
+      - name: Publish new release of @manifoldco/mui-core 🚀
+        run: 'npm publish --tag latest'
```

Comment:
How are we setting the version number? I assumed we'd be updating the `version` field in `package.json` with the tag, but I don't see that happening. Did I miss it?

Reply:
That happens in the `prepare` script, which happens on `npm install` and `npm publish`. Putting it there makes sure it happens every time, both in CI or if someone publishes manually